quasardb/pandas/__init__.py

```diff
@@ -253,11 +275,18 @@ def write_dataframe(df, cluster, table, create=False, _async=False, chunk_size=5
                     fn = write_with[ct]
                     fn(i, v)
 
+        start = time.time()
+
+        logger.debug("push chunk of %d rows, fast?=%s, async?=%s", len(current_df.index), fast, _async)
+
+        if fast is True:
+            batch.push_async()
```

Comment:
Didn't you mean:
```suggestion
            batch.push_fast()
```
?